app/Resources/static/themes/_global/index.js

```diff
@@ -29,7 +29,20 @@ $(document).ready(() => {
     });
 
     const x = JSON.parse($('#annotationroutes').html());
-    app.include(annotator.storage.http, x);
+    app.include(annotator.storage.http, $.extend({}, x, {
+      onError(msg, xhr) {
+        if (!Object.prototype.hasOwnProperty.call(xhr, 'responseJSON')) {
+          return;
```

Comment:
No need for translations, since it's an error message and should not happen often. Moreover all of this should be changed at some point, since https://web.hypothes.is/blog/annotation-is-now-a-web-standard/